frontend/src/static/css/styles.scss

```
@@ -1621,7 +1621,7 @@ video.show-video {
 	width: 100%;
 	height: 100%;
 
-	background: #000;
+	background: var(--cinemata-black);
```

Stylelint (17.11.1)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)